### PR TITLE
Tech Docs review of packedBERT/packedBERT_single_label_text_classification.ipynb

### PR DESCRIPTION
**Pull request by jayniep-gc**
This PR generally improves the text in the notebook. The following have been done:
1. Title made consistent with template
2. Added next steps
3. Made environment setup easily removable
4. Added summary table and button for joining Slack
5. Made text clearer

### CODE REVIEW

File: notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb [hunk 1]
```diff
@@ -1,200 +1,158 @@
...
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's start by importing and printing out the versions of `Transformers` and `Optimum Graphcore`:"
+    "|  Domain | Tasks | Model | Datasets | Workflow |   Number of IPUs   | Execution time |\n",
+    "|---------|-------|-------|----------|----------|--------------|--------------|\n",
+    "| Natural language processing | Text classification | PackedBERT | GLUE Benchmark | Fine-tuning |  |   |"
```
Comment (evawGraphcore) [inline notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb:66]: We only have results for our latest testing locally so I dont think they will be reliable enough to use

File: notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb [hunk 2]
```diff
@@ -735,17 +756,18 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In order to pack efficiently, we will use a histogram-based algorithm: shortest-pack-first histogram packing (SPFHP) presented in the [blog post](https://www.graphcore.ai/posts/introducing-packed-bert-for-2x-faster-training-in-natural-language-processing) adapted from the [blog code](https://github.com/graphcore/tutorials/tree/master/blogs_code/packedBERT). The full process of packing the dataset consists of four steps:\n",
+    "In order to pack efficiently, we will use a histogram-based algorithm. The shortest-pack-first histogram packing (SPFHP) was presented in the Graphcore blog post [introducing Packed BERT for a training speedup in natural language processing](https://www.graphcore.ai/posts/introducing-packed-bert-for-2x-faster-training-in-natural-language-processing). We have adapted the [code](https://github.com/graphcore/tutorials/tree/master/blogs_code/packedBERT) from the blog post for this notebook. The full process of packing the dataset consists of four steps:\n",
```
Comment (evawGraphcore) [inline notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb:763]: tutorials is now in examples: https://github.com/graphcore/examples/tree/master/tutorials/blogs_code/packedBERT

File: notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb [hunk 3]
```diff
@@ -888,35 +913,38 @@
...
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {
     "id": "545PP3o8IrJV"
    },
    "source": [
-    "## Fine-tuning the model"
+    "# Fine-tuning the model"
```
Comment (evawGraphcore) [inline notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb:922]: I dont think we want this to be a main header

File: notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb [hunk 4]
```diff
@@ -888,35 +913,38 @@
...
+    "Now that our data is ready, we can download the pre-trained model and fine-tune it."
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Implement Packed BERT\n",
+    "## Implement Packed BERT\n",
```
Comment (evawGraphcore) [inline notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb:940]: I think this should keep ###

File: notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb (hunk 2 above)
Comment (jayniep-gc) [reply notebooks/packed_bert/packedBERT_single_label_text_classification.ipynb:763]: Will be addressed in a separate PR.

**Review by evawGraphcore — APPROVED** [review]
LGTM thanks

**Pull request merged**